plugins/admission/pkg/admission/validate.go

```diff
@@ -135,20 +160,24 @@ func (v *Validator) Handle(ctx context.Context, req admission.Request) admission
 	return response
 }
 
-func getRequestReport(req admission.Request) (models.ReportInfo, error) {
-	report := models.ReportInfo{
-		Report:  "metadata",
-		Version: "0.1.0",
-	}
-	var err error
-	report.Contents, err = json.Marshal(&req.AdmissionRequest)
-	return report, err
+type MetadataReport struct {
+	admissionv1.AdmissionRequest
+	NamespaceObject map[string]any `json:"namespaceObject,omitempty"`
 }
 
-func processInputYAML(ctx context.Context, configurationObject models.Configuration, input []byte, decodedObject map[string]interface{}, token, name, namespace, kind, apiGroup string, metaReport models.ReportInfo) (bool, []string, []string, error) {
-	reports := []models.ReportInfo{
-		metaReport,
-	}
+func getRequestReport(req admission.Request, namespaceObject map[string]any) (models.ReportInfo, error) {
+	metadataReport := MetadataReport{AdmissionRequest: req.AdmissionRequest, NamespaceObject: namespaceObject}
+	contents, err := json.Marshal(&metadataReport)
+	return models.ReportInfo{
+		Report:   "metadata",
+		Version:  "0.2.0",
```

Comment:
I bump the version to `0.2.0` as we are adding a new field to it... or `0.1.1` would be enough?

Reply:
I don't think we really use this version, so this is fine